### ISSUE TITLE
Account menu lists four items inside but only three items are available (Accessibility)

### ISSUE THREAD

**Issue by tofumatt** [google/site-kit-wp#6636]
## Bug Description

When the accounts menu item is opened it lists four (4) items, but only two (3) are available to cycle through/select with the keyboard. This is probably a result of extra child DOM elements that aren’t interactive.

## Steps to reproduce

1. Enable a screen reader (VoiceOver on macOS in the example)
2. Focus the user account menu and open it with a keyboard
3. Notice the menu is announced and said to have 4 items, but only 3 should exist (the first is the account info, the other two are links/buttons to "manage site" and "disconnect")

## Screenshots

https://user-images.githubusercontent.com/90871/221062509-7d685609-0211-4db8-9670-e67e023899a8.mp4

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* The number of menu items read aloud to a screen reader when the user menu is opened should match the number of items that can be moved through and appear visually. (In this case: 3)

## Implementation Brief

* In `assets/js/components/UserMenu/Details.js`
  * Add a `aria-label="Google account"` to the wrapping `div` (with i18n support)
  * Add `aria-label="Email"` to the email paragraph (with i18n support)
  * Set the alt tag of the profile picture to an empty value, so it's ignored by screenreaders. This corresponds to the WCAG ["decorative images" recommendation](https://www.w3.org/WAI/tutorials/images/decorative/).
  

### Test Coverage

* No test coverage needed.

## QA Brief

* Follows steps to reproduce and ensure that it works correctly now.

## Changelog entry

* Update Google account menu's accessibility labels to improve VoiceOver navigation.

**Comment by derweili** [comment]
The screenreader refers to the following 4 elements:

- The profile picture
- The account information (name and email)
- "Manage Sites" menu item
- "Disconnect" menu item

I would recommend to hide the profile for screen readers as this is just decorative. After that, the screenreader would only list 3 items which is correct as it says there are 3 items, not 3 menu-items. 

The accessibility issues for not labeling the account information correctly is adressed in #6637

**Comment by derweili** [comment]
>  but only two (2) are available to cycle through/select with the keyboard.
This is not 100% correct. 

When using tab navigation, you can access only 2, thats correct. But tab navigation only take "clickable" elements into account. Therefore non clickable elements can't be accessed.

When using the mac voiceover control (ctrl + shift + left/right-arrow) you can access all 4 elements.

**Comment by eugene-manuilov** [comment]
IB ✔️

**Comment by wpdarren** [comment]
### QA Update: ⚠️ 

@tofumatt in the AC it refers to 2 items, but title suggests it should be 3 items which is what I am seeing.  I am wondering if the 3 items is correct though because as you get to the account menu, and use arrow keys on keyboard there are only two items that are selectable in the screen reader (Voice over).  Maybe the 3rd item is the account name/avatar, but this isn't selectable to the user.  Interested in your thoughts.

**Comment by tofumatt** [comment]
@wpdarren 100% yes! I actually updated all of the issue text to say three instead of two… except in that one spot 😅 

Three is correct, so if that's what you saw it should be good for QA 👍🏻

**Comment by wpdarren** [comment]
### QA Update: ✅ 

**Verified:**

- There are 3 items when using `VoiceOver` as per the AC.  See screencast below.

https://github.com/google/site-kit-wp/assets/73545194/ac78a822-2dc4-4b71-89d4-78527df80a2a
